.laminas-ci.json

```diff
@@ -0,0 +1,7 @@
+{
+  "exclude": [
+    {
+      "name": "Infection"
```

Comment:
Eh, let's not exclude mutation testing

Reply:
not working due to coverage not available ... :'(

Reply:
see https://github.com/laminas/laminas-continuous-integration-action/issues/130

Reply:
Urgh, that thing again: will not have time for it in the near future :-( 

Sorry, but not moving forward by disabling mutation testing, so it will have to wait.